bindings/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+mod bindings {
+    wit_bindgen::generate!({
+        path: "../wit",
+        world: "reactor",
+        std_feature,
+        // The generated definition of command will pull in std, so we are defining it
+        // manually below instead
+        skip: ["command", "get-preopens", "get-environment"],
```

Comment:
I don't think we want to skip these bindings - thats a special case in the adapter